packages/web-twig/src/Resources/components/Stack/StackItem.twig

```diff
@@ -0,0 +1,23 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _elementType = props.elementType | default('div') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'StackItem' -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _classNames = [ _rootClassName, _styleProps.className ] -%}
+
+{%- set _style = '' -%}
+
+{# Attributes #}
+{%- set _styleAttr = _style or (_styleProps.style is not same as(null)) ? 'style="' ~ _style ~ _styleProps.style | join() ~ '"' -%}
```

Comment:
why? We don't do this for any other component. Why not just use `{{ styleProp(_styleProps) }}` on the main element?